src/Extensions/ServiceFabricGuest.Abstractions/SfConfigurationProvider.cs

```diff
@@ -2,22 +2,35 @@
 // Licensed under the MIT license.
 
 using System;
-using System.Globalization;
-
-namespace Microsoft.Omex.Extensions.Hosting.Services.Web
-{
-	internal class SfConfigurationProvider
+
+namespace Microsoft.Omex.Extensions.ServiceFabricGuest.Abstractions
+{
+	/// <summary>
+	/// Configuration provider for service fabric environment variables
+	/// </summary>
+	public class SfConfigurationProvider
```

Comment:
Is this class designed to be used only from guest executables?

Reply:
Generally it can be used from any type of service, especially from guest executables due to lack of dependency to service context

Reply:
We also use it to create WebService with a single host, because we need to provide endpoint port to Kestrel before ServiceContext available. I think in the future we would move all interactions with SF environment variables into it.